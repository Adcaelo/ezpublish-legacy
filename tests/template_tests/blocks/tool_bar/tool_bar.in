{tool_bar name='top' view='line'}
