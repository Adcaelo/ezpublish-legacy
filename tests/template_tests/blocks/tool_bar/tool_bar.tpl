{tool_bar name='top' value='view'}
