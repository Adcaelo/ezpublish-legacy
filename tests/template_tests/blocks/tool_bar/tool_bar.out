
<div class="toolbar-item first">
    <div class="toolbox">
        <div class="toolbox-design">
                    <h2>Not logged in</h2>
        
        <div class="toolbox-content">
                <form method="post" action="/user/login">
        <div class="info-text">
            <label for="id1">Username</label>
            <input class="textinput" type="text" size="10" name="Login" id="id1" value="" />
        </div>

        <div class="info-text">
            <label for="id2">Password</label>
            <input class="textinput" type="password" size="10" name="Password" id="id2" value="" />
        </div>

        <input type="image" class="searchimage" src="/design/standard/images/1x1.gif" />
        <div class="content-link">
            <p>
                <a href="/user/register">Not registered?</a>
            </p>
        </div>
        <div class="content-link">
            <p>
            <a href="/user/forgotpassword">Forgot your password?</a>
            </p>
        </div>
        </form>
                </div>

        </div>
    </div>
</div>
